types/react-native-elements/index.d.ts

```diff
@@ -1,11 +1,51 @@
-// Type definitions for react-native-elements 0.13
+// Type definitions for react-native-elements 0.16
 // Project: https://github.com/react-native-training/react-native-elements#readme
 // Definitions by: Kyle Roach <https://github.com/iRoachie>
+//                 Ifiok Jr. <https://github.com/ifiokjr>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.3
 
 import * as React from 'react';
-import { ViewStyle, TextStyle } from 'react-native';
+import {
+    ViewStyle,
+    TextStyle,
+    ImageStyle,
+    ImageURISource,
+    TouchableWithoutFeedbackProps,
+    TouchableHighlightProperties,
+    TouchableOpacityProperties,
+    ViewProperties,
+    TextInputProperties,
+    TextInput,
+    StatusBarProperties,
+    KeyboardType,
+    KeyboardTypeIOS,
+    StyleProp,
+} from 'react-native';
+
+export type IconType =
```

Comment:
This restricts custom iconSets the user may implement. See https://github.com/oblador/react-native-vector-icons#custom-fonts